sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpConnectionScope.cs

```diff
@@ -754,6 +754,11 @@ protected virtual TimeSpan CalculateLinkAuthorizationRefreshInterval(DateTime ex
 
                 try
                 {
+                    if (refreshTimer == null)
```

Comment:
Note:  This will help reduce the potential for occurrence, but does not eliminate it as the `Dispose` call could be taking place as the timer callback is invoked.